### ISSUE TITLE
column-coordinates greater than Z are not working

### ISSUE THREAD

**Issue by netpsirit72** [PHPOffice/PhpSpreadsheet#546]
This is a bug report!

Column-coordinates greater than Z are not working

```php
<?php
require __DIR__ . '/vendor/autoload.php';
// Create new Spreadsheet object
$spreadsheet = new \PhpOffice\PhpSpreadsheet\Spreadsheet();
...
$activeSheet->mergeCells("Z1:AA1");
```

### Which versions of PhpSpreadsheet and PHP are affected?
1.3.0

**Comment by PowerKiKi** [comment]
Duplicate of #545 and #521 latest comments

**Comment by PowerKiKi** [comment]
It's been released as [1.3.1](https://github.com/PHPOffice/PhpSpreadsheet/releases/tag/1.3.1)
